src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-best-practices-resources.mdx

```diff
@@ -78,7 +74,7 @@ for monitoring host entities using the host metrics receiver for the OpenTelemet
 
 #### Required attributes [#required-attributes]
 
-* [`host.name`](https://opentelemetry.io/docs/specs/semconv/attributes-registry/host)
+* [`host.name`](https://opentelemetry.io/docs/specs/semconv/attributes-registry/host): The host name you see in the UI comes from this value.
 * [`host.id`](https://opentelemetry.io/docs/specs/semconv/attributes-registry/host)
```

Comment:
@alanwest Can you also add a description for host.id, for consistency in this section?